### PR TITLE
feat(kubernetes-ingress): Add Values.controller.extraEnvFrom to deployment and daemonset

### PR DESCRIPTION
**Pull request by mecampbellsoup**
Analogous to https://github.com/haproxytech/helm-charts/pull/252, but for `kubernetes-ingress` chart.

### CODE REVIEW

**Comment by mecampbellsoup** [comment]
@dkorunic 😄

File: kubernetes-ingress/templates/controller-deployment.yaml [hunk 1]
```diff
@@ -206,7 +206,7 @@ spec:
               fieldRef:
                 fieldPath: status.podIP
           {{- if .Values.controller.extraEnvs -}}
-          {{- toYaml .Values.controller.extraEnvs | nindent 10 }}
+            {{- toYaml .Values.controller.extraEnvs | nindent 12 }}
```
Comment (dkorunic) [inline kubernetes-ingress/templates/controller-deployment.yaml:209]: Can you revise this please?

Comment (mecampbellsoup) [reply kubernetes-ingress/templates/controller-deployment.yaml:209]: As in revert?

Comment (mecampbellsoup) [reply kubernetes-ingress/templates/controller-deployment.yaml]: 12, not 10, is correct value right? I will revert the indent if that's what you mean though.

**Comment by mecampbellsoup** [comment]
@dkorunic ready!

**Comment by dkorunic** [comment]
Looks good, merging.

**Comment by mecampbellsoup** [comment]
Thank you @dkorunic 😄 do you know when the next release tag is scheduled to be published that includes this? 🙏

**Pull request merged**